lib/pmon.c

```diff
@@ -60,29 +60,29 @@
 #define ENTRY(x, h) {.mask=x, .name=#x, .help=h}
 
 const struct switchtec_evcntr_type_list switchtec_evcntr_type_list[] = {
-	ENTRY(ALL, "All Events"),
-	ENTRY(ALL_TLPS, "All TLPs"),
-	ENTRY(ALL_ERRORS, "All errors"),
+	ENTRY(ALL, "all events"),
+	ENTRY(ALL_TLPS, "all TLPs"),
+	ENTRY(ALL_ERRORS, "all errors"),
 	ENTRY(UNSUP_REQ_ERR, "Unsupported Request error"),
 	ENTRY(ECRC_ERR, "ECRC error"),
 	ENTRY(MALFORM_TLP_ERR, "Malformed TLP error"),
-	ENTRY(RCVR_OFLOW_ERR, "Receiver overflow error"),
+	ENTRY(RCVR_OFLOW_ERR, "Receiver Overflow error"),
 	ENTRY(CMPLTR_ABORT_ERR, "Completer Abort error"),
 	ENTRY(POISONED_TLP_ERR, "Poisoned TLP error"),
-	ENTRY(SURPRISE_DOWN_ERR, "Surprise down error"),
-	ENTRY(DATA_LINK_PROTO_ERR, "Data Link protocol error"),
+	ENTRY(SURPRISE_DOWN_ERR, "Surprise Down error"),
+	ENTRY(DATA_LINK_PROTO_ERR, "Data Link Protocol error"),
 	ENTRY(HDR_LOG_OFLOW_ERR, "Header Log Overflow error"),
-	ENTRY(UNCOR_INT_ERR, "Uncorrectable Internal error"),
-	ENTRY(REPLAY_TMR_TIMEOUT, "Replay timer timeout"),
-	ENTRY(REPLAY_NUM_ROLLOVER, "Replay number rollover"),
+	ENTRY(UNCOR_INT_ERR, "Uncorrectable Internal Error"),
```

Comment:
Error->error

Reply:
I upper-cased it here as "Uncorrectable Internal Error" is a standard term used in the PCIe spec.